python/lsst/ip/isr/ptcDataset.py

```diff
@@ -569,6 +597,13 @@ def fromTable(cls, tableList):
                 inDict['histVars'][ampName] = record['HIST_VARS']
                 inDict['histChi2Dofs'][ampName] = record['HIST_CHI2_DOFS']
                 inDict['kspValues'][ampName] = record['KS_PVALUES']
+            if calibVersion == 1.3:
```

Comment:
This should be `calibVersion >= 1.3`.  And I realized that above it should be `if calibVersion < 1.2`.  Or perhaps for better parallelism, this new code should be reversed, and `calibVersion < 1.3` should be checked to set to nans, and else is to use the noise matrix.

Reply:
I just changed to `calibVersion >= 1.3` and `if calibVersion < 1.2`, as you suggest.  And the docstring was missing the comment for v1.2; I added "Version 1.2 adds the `histVars`, `histChi2Dofs`, and `kspValues`
    attributes.". Please let me know if this is OK.